### PR TITLE
loading datasets from DATASETSROOT directory

### PR DESCRIPTION
**Pull request by dmitrii-kriukov**
if DATASETSROOT environment variable assigned, load datasets from assigned location

### CODE REVIEW

**Comment by dmitrii-kriukov** [comment]
http://intel-ci.intel.com/ec82c791-9275-f126-b310-8cdcd4b723a1

**Review by vlad-nazarov — APPROVED** [review]

File: runner.py [hunk 1]
```diff
@@ -84,8 +83,15 @@ def get_configs(path: Path) -> List[str]:
...
-    os.makedirs('data', exist_ok=True)
+    env = os.environ.copy()
+    if 'DATASETSROOT' in env:
+        datasets_root = env['DATASETSROOT']
+        logging.info(f'datasets folder at {datasets_root}')
+    elif 'DAAL_DATASETS' in env:
+        datasets_root = env['DAAL_DATASETS']
+        logging.info(f'datasets folder at {datasets_root}')
+    else:
+        datasets_root = ''
```
Comment (Alexsandruss) [inline runner.py:94]: Add logging info: 'datasets folder not found'

File: runner.py [hunk 2]
```diff
@@ -155,23 +161,41 @@ def get_configs(path: Path) -> List[str]:
...
+                    if dataset_path is None:
+                        logging.warning(
+                            f'Dataset {dataset_name} could not be loaded. \n'
+                            'Check the correct name or expand the download in '
+                            'the folder dataset.'
+                            )
+                        continue
+                    elif not dataset_path and datasets_root:
+                        logging.info(
+                            f'{dataset_name} is taken from  local folder'
```
Comment (Alexsandruss) [inline runner.py:192]: double space

File: runner.py [hunk 3]
```diff
@@ -84,8 +83,15 @@ def get_configs(path: Path) -> List[str]:
         stream=sys.stdout, format='%(levelname)s: %(message)s', level=args.verbose)
     hostname = socket.gethostname()
 
-    # make directory for data if it doesn't exist
-    os.makedirs('data', exist_ok=True)
+    env = os.environ.copy()
+    if 'DATASETSROOT' in env:
+        datasets_root = env['DATASETSROOT']
+        logging.info(f'datasets folder at {datasets_root}')
```
Comment (Alexsandruss) [inline runner.py:89]: logging message should start from capital letter

**Review by Alexsandruss — APPROVED** [review]

**Pull request merged**